sirepo/package_data/static/json/flash-schema.json

```diff
@@ -1293,36 +1786,123 @@
             "hx_dtFactor": ["hx_dtFactor", "Float", 0.5, "", 0.0],
             "hx_ieTimeCoef": ["hx_ieTimeCoef", "Float", 1.0, "Constant coefficient for scaling ion/ele coupling time", 0.0],
             "hx_relTol": ["hx_relTol", "Float", -1.0, "relative tolerance for temperature errors introduced by HeatExchange.\nThis runtime parameter affects the time step computed by\nHeatexchange_computeDt. Basically, if the max (abs) temperature\nadjustment that would be introduced in any nonzero component in any cell\nis less than hx_relTol, then the time step limit is relaxed. Set to a\nnegative value to inherite the value of runtime parameter eos_tolerance."]
-        }
-    },
-    "view": {
-        "gridEvolutionAnimation": {
-            "title": "Grid Quantity Evolution",
-            "advanced": [
-                "y1",
-                "y2",
-                "y3",
-                "notes"
-            ]
         },
-        "varAnimation": {
-            "title": "Variable Plot",
-            "advanced": [
-                "var",
-                "framesPerSecond",
-                "colorMap",
-                "notes"
-            ]
+        "setupArguments": {
```

Comment:
@moellep this and the referenced enums and corresponding view are the only relevant part of this diff. The rest of the changes are just sorting everything alphabetically.